extension-manifest-v3/src/store/utms.js

```diff
@@ -0,0 +1,18 @@
+import { store } from 'hybrids';
```

Comment:
Is it something that will be used in hybrids components contexts at all? I was thinking about not using the store for both introduced models.

Reply:
ok, make sense